hack/deploy.sh

```diff
@@ -5,7 +5,8 @@ set -euxo pipefail
 source "$(dirname $0)/common"
 
 if [ $REMOTE_REGISTRY = false ] ; then
-    oc create -n ${NAMESPACE} -f manifests/05-deployment.yaml
+    $repo_dir/hack/gen-olm-artifacts.sh ${CSV_FILE} ${NAMESPACE} 'dep' \
+    |  oc create -f -
```

Comment:
if you ran deploy-setup.sh it would create the `dep` already - will it cause an error if the `dep` already exists?

Reply:
looks like it would.  For equivalence I will remove `dep` from `deploy-setup` which should make it equivalent